ksqldb-functional-tests/src/test/resources/query-validation-tests/insert-into.json

```diff
@@ -137,6 +137,42 @@
         {"topic": "sink", "key": "0", "value": {"A": 789, "B": "turtle"}, "timestamp": 0}
       ]
     },
+    {
+      "name": "convert formats: JSON to PROTOBUF",
```

Comment:
Do these tests fail prior to the changes in this PR? I don't see them using nested structs so I'm not sure how they're related to the bugfix in this PR.

Reply:
I just added these tests to have some INSERT INTO coverage with protobuf. I was running INSERT INTO manually, so wanted to automated them.